### ISSUE TITLE
Riss does not biuld a .so library for riss+coprocessor as required for IPASIR

### ISSUE THREAD

**Issue by nmanthey** [nmanthey/riss-solver#10]
When using dynamic linking against the IPASIR interface, two libraries would have to be used for Riss, as Coprocessor comes in its own library.

Similarly to the static library, add a rule to build a dynamic library, and build it when IPASIR is build.

**Comment by kahlertl** [comment]
Hi @nmanthey, it's me again :smiley: Long time no see!

Isn't there the `riss-coprocessor-lib-shared` target when the `FPIC` option is enabled (see [CMakeLists.txt#L150](https://github.com/nmanthey/riss-solver/blob/f49afd2e624d58540b5a313f794392d386fd33a4/CMakeLists.txt#L150))? This is a combined shared library of `riss` and `coprocessor`.

**Comment by conp-solutions** [comment]
Yes, true. This file is generated: ./lib/libriss-coprocessor.so. Let's close this.
